### PR TITLE
[For Review] PR for ingestion (#3)

### PR DESCRIPTION
**Pull request by PaulKinlan**
This PR is to open up the Ingest tool for review, critique etc to see if it's the right fit and shape for the project.

+ CLI - `@polymath/ingest --openai-api-key=[SECRET] rss https://paul.kinlan.me/index.xml` saves to a file in the correct format
  + Right now there is a test harness and one ingest library 'rss' as an example.
  + You can have external ingest modules (`@polymath/ingest --openai-api-key=[SECRET] ../../rss-module.js https://paul.kinlan.me/index.xml`
+ Library - ``
  + You can create custom importers `class X extends Importer`
  + You can embed the entire algorithm in your own library (dedicated tool to export to pinecone)  
    ``` 
    import { Importer } from "./importer.js";

    const importer = new Importer({ /* options */ });

    for await(const bit of importer.run({ args, options, command })) { 
      console.log(`${bit.url}: Embedding: ${bit.embedding}`);
      // Save to embedding to pinecone... or batch later.   
    }
    ```

TODO: Add more tests. Does it extract the data in the same way as the python library? (i.e, min tokens and max-tokens)

### CODE REVIEW

File: .vscode/extensions.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+	// See https://go.microsoft.com/fwlink/?LinkId=827846 to learn about workspace recommendations.
+	// Extension identifier format: ${publisher}.${name}. Example: vscode.csharp
+
+	// List of extensions which should be recommended for users of this workspace.
```
Comment (dglazkov) [inline .vscode/extensions.json]: Probably don't want this file, right?

File: core/ingest/README.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
+# Polymath Ingest
+
+This is a tool that is used to ingest data from a source and convert it into a format that can be used by the Polymath system.
+
+## Standalone ingestion tool
```
Comment (dglazkov) [inline core/ingest/README.md:5]: Can this go into `core/cli`? In other words, most of the importing code stays here, but the actual invocation is made using `polymath` tool. I'd love to keep only one CLI entry point for Polymath.

**Review by dglazkov — APPROVED** [review]
This is great, modulo comments.

File: .vscode/extensions.json (hunk 1 above)
Comment (PaulKinlan) [reply .vscode/extensions.json]: Yeah - I will remove.

File: core/ingest/README.md (hunk 2 above)
Comment (PaulKinlan) [reply core/ingest/README.md:5]: I was thinking about this a bit the other day and I was worrying that the base CLI would get incredibly complex... however, I could do an "import" argument.

**Comment by PaulKinlan** [comment]
@dglazkov Removed the independent CLI and migrated it to the existing CLI.

**Review by dglazkov — APPROVED** [review]
LGTM!

**Pull request merged**